book/website/ethical-research/data-hazards/dh-case-study.md

```diff
@@ -0,0 +1,281 @@
+
+(er-datahazardscasestudy-hosting-hybridevent)=
+### Hybrid Event Tips
+We chose to host the Symposium in a hybrid format as this would allow attendance for those interested in the topic, and speakers who would be unable to attend in-person.
+
+Thanks to the good communication with volunteers and the preparation of the room before the event, our hybrid format ran successfully without hiccups, even though this aspect of the event was challenging due to the number of tasks involved. Here we share some tips that helped us!
+
+To ensure a smooth hybrid event, we:
+- tested the physical room audio-visual the day before and the day of the event, to ensure there were multiple microphones and the capability for live Zoom recordings.
+- had in-person session chairs to introduce speakers and make sure people (both online and in-person) kept to agenda timings.
+- made sure session chairs were aware of the hybrid set-up, including microphones, speakers and Zoom settings,
+- made sure that there was always at least one person in the room who was comfortable with the hosting technology.
+- set a non-disruptive 1min warning, using a 1min sign to hold up in-person, or type into the chat online, to signal the end of the talk.
+- kept questions to the end of the talks to avoid disruptions and talks becoming longer.
+- kept in contact with online hosts throughout the day to ensure both the presentations and the audio were working.
+- had our online volunteers keep track of break-out rooms, host discussions, and keep track of questions for Q&A sessions.
+
+
+(er-datahazardscasestudy-hosting-accessibility)=
+### Accessibility
+Below we provide some insight into what was important for us to consider during the event.
+
+**Code of Conduct**
+We put together a code of conduct for the event that reflected our values and the values of the community we aimed to create during the symposium. It also summarised the expectations of participants joining the symposium and laid out reporting guidelines and contact points for if the code of conduct was breached during the event.
+
+Please find our Code of Conduct [here](https://hackmd.io/@7D_si7-qQwKdepUrj7_AzA/DER_CoC).
+
+Additionally, overall, below we highlight some points that we actioned to ensure that the hybrid format and the event were accessible:
+
+- collected accessibility requirements at the time of advertising, to help with planning.
+- used [Otter.ai](https://otter.ai/), a speech-to-text transcription application using artificial intelligence and machine learning. 
+  It shows captions for live speakers and generates written transcriptions. 
+  Note: Otter.ai is not always sufficient as it has been trained on English and American accents, so further editing of subtitles or transcripts might be needed.
+- ensured all participants in-person were using microphones when speaking.
+  Microphones can be helpful for all hearing requirements, both online and in-person (don't assume everyone can hear equally).
```

Comment:
Thank you for pointing this out! I always have trouble motivating people to use them :/